pallets/identity/src/lib.rs

```diff
@@ -2049,6 +2049,77 @@ impl<T: Trait> Module<T> {
             RpcDidRecords::IdNotFound
         }
     }
+
+    /// Use to get the filtered link data for a given signatory
+    /// - if link_type is None then return links data on the basis of the `allow_expired` boolean
+    /// - if link_type is Some(value) then return filtered links on the value basis type in conjunction
+    ///   with `allow_expired` boolean condition
+    pub fn get_filtered_links(
+        signatory: Signatory,
+        allow_expired: bool,
+        link_type: Option<LinkType>,
+    ) -> Vec<Link<T::Moment>> {
+        let now = <pallet_timestamp::Module<T>>::get();
+
+        if let Some(type_of_link) = link_type {
+            <Links<T>>::iter_prefix(signatory)
```

Comment:
Unbounded operation, Is the number of links limited?

Reply:
It's an "offchain" call so it's ok. We have plenty of unbounded operations in rpc.